js/app.js

```diff
@@ -100,24 +66,23 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
             }
             return url;
         }
-
-    //Create url for altcoin payment
-    $scope.alt_track_url = function(uuid) {
-        var params = getParameterByNameBlocko('wc-api');
-        if (params)
-            params = {
-                "wc-api": params
-            };
-            else
-                params = {};
-            params.uuid = uuid;
-            url = window.location.pathname;
-            var serializedParams = $httpParamSerializer(params);
-            if (serializedParams.length > 0) {
-                url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
-            }
-            return url;
+    
+    // Select Blockonomics currency
+    $scope.select_blockonomics_currency = function(blockonomics_currency) {
+        $scope.currency = blockonomics_currency;
+        $scope.currency_selecter  = false;
+    }
+
+    Order.get({
```

Comment:
This can be done before loading the page and then passing the active currencies to the scope using something like this: 
```
<div id="active_currencies" data-active_currencies='<?php echo $active_currencies; ?>'></div>
```
and
```
    var active_currencies = JSON.parse(active_currencies_div.dataset.active_currencies);
    $scope.active_currencies = active_currencies;
```
This will save on an additional call from the client side to server side.